speechbrain/core.py

```diff
@@ -965,11 +969,17 @@ def fit(
             Whether to display the progress of each epoch in a progressbar.
         """
 
-        if not isinstance(train_set, DataLoader):
+        if not (
+            isinstance(train_set, DataLoader)
+            or isinstance(train_set, LoopedLoader)
+        ):
```

Comment:
Is there no way to make the `LoopedLoader` inherit from `DataLoader`? It seems like you should only need one check for these...

Reply:
Not really, DataLoader is very complex and this really does a different thing.